Celeste.Mod.mm/Mod/Registry/DecalRegistry.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
```

Comment:
this (and `System.IO`) are unused:
```suggestion
```
(github won't let me suggest the other deletion)